doc/examples/Repository.rst

```diff
@@ -8,4 +8,43 @@ Get repository topics
 
     >>> repo = g.get_repo("PyGithub/PyGithub")
     >>> repo.get_topics()
-    [u'pygithub', u'python', u'github', u'github-api']
\ No newline at end of file
+    [u'pygithub', u'python', u'github', u'github-api']
+
+Get count of stars
+------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> repo.stargrazers_count
+    2086
+
+Get list of open issues
+--------------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> open_issues = repo.get_issues(state='open')
+    >>> for issue in open_issues:
+    ...    print(issue.title)
+    ...
+    How to get public events?
+	GithubException: 500 None when using get_issues
```

Comment:
Seems like caused by #911
But we don't want this to appear on the example, don't we?

Reply:
:)

Reply:
@sfdye Removed the exception issue in the example